src/lib.rs

```diff
@@ -118,13 +194,16 @@ impl Mapping {
     ///
     /// If the `Mapping` was constructed with `with_functions`, information about the containing
     /// function may also be returned when available.
-    pub fn locate(&self, addr: u64) -> Option<(path::PathBuf, u64, Option<Cow<str>>)> {
-        self.inner.locate(addr)
+    pub fn locate(&self, addr: u64) -> Option<(path::PathBuf, Option<u64>, Option<Cow<str>>)> {
+        self.inner.locate(addr).unwrap_or_else(|_| {
+            // TODO: we probably want to log this, or assert failure when testing?
```

Comment:
I discussed this briefly in the comments on the PR above, but it is basically to expose a simpler API to the user. In general, they probably want errors to simply return `None` as the location, rather than having to do error matching themselves. That said, forcing the user to do `unwrap_or(None)` isn't all that heavy a burden. I guess I can change it.